packages/opentelemetry/README.md

```diff
@@ -212,6 +263,24 @@ setupOpenTelemetry({
 
 **Deprecated**. This will still work but has been replaced with [`options.tracing.authorizationHeader`](#optionstracingauthorizationheader), which is now the preferred way to set this option.
 
+#### `options.metrics`
+
+An object containing other metrics-specific configurations. Defaults to `undefined` which means that OpenTelemetry metrics will not be sent.
+
+#### `options.metrics.endpoint`
+
+A URL to send OpenTelemetry metrics to. E.g. `http://localhost:4318/v1/metrics`. Defaults to `undefined` which means that OpenTelemetry metrics will not be sent.
+
+**Environment variable:** `OPENTELEMETRY_METRICS_ENDPOINT`<br/>
+**Option:** `metrics.endpoint` (`String`)
+
+#### `options.metrics.apiGatewayKey`
+
+Set the `Authorization` HTTP header in requests to the central API-Gateway-backed OpenTelemetry metrics collector. Defaults to `undefined`.
+
+**Environment variable:** `OPENTELEMETRY_API_GATEWAY_KEY`<br/>
```

Comment:
**question:** I wondered if it was worth naming this `OPENTELEMETRY_METRICS_API_GATEWAY_KEY`. It really depends whether we're likely to be able to use the same key in future if tracing support is added centrally. I imagine the answer is "not sure" but @sjparkinson do you think metrics/tracing will require different keys in future?

Reply:
The same key and endpoint will be used for traces when we eventually get around to supporting that centrally.

Reply:
`OTEL_EXPORTER_OTLP_API_KEY` is what we have used in docs to align with the naming convention of other environment variables used by the SDKs.

Strictly speaking we could/should? be using [`OTEL_EXPORTER_OTLP_HEADERS`](https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/#otel_exporter_otlp_headers), but the format of the value doesn't work well with the API Gateway's Doppler integration.

Reply:
Ah yeah about those, we're warning people off them because they can mess with the way we've configured everything. It's fine to use `OTEL_` vars but we don't want to have to support you if they break something 😂

https://github.com/Financial-Times/dotcom-reliability-kit/blob/d49b41d02292d7d0a1201d43fa406f0355500936/packages/opentelemetry/lib/index.js#L59-L69

This is why we went with our own naming instead of the one in your docs.